iree/tools/iree-e2e-matmul-test.c

```diff
@@ -526,6 +528,53 @@ static iree_status_t copy_list_of_buffer_views(
   return iree_ok_status();
 }
 
+static iree_status_t iree_cpu_features_have_all_target_features(
+    iree_cpu_features_t* cpu_features, yaml_document_t* document,
+    yaml_node_t* target_features_node) {
+  for (yaml_node_item_t* item = target_features_node->data.sequence.items.start;
+       item != target_features_node->data.sequence.items.top; ++item) {
+    yaml_node_t* item_node = yaml_document_get_node(document, *item);
+    iree_string_view_t required_feature = iree_yaml_node_as_string(item_node);
+    if (iree_string_view_is_empty(required_feature)) continue;
+    bool feature_is_supported = false;
+    IREE_RETURN_IF_ERROR(iree_cpu_features_query(cpu_features, required_feature,
+                                                 &feature_is_supported));
+    if (!feature_is_supported) {
+      return iree_make_status(
+          // The error status matters. We distinguish "feature not supported",
+          // which is a normal thing to happen, from actual errors.
+          IREE_STATUS_CANCELLED,
```

Comment:
this should be a IREE_STATUS_UNAVAILABLE - CANCELLED indicates user-cancelation

Reply:
ok